### DISCUSSION TITLE
A problem with the Arabic version

### DISCUSSION THREAD

**Discussion by Mohammed1948** [Stellarium/stellarium#3247] [Q&A]
Hi, I have a problem with the Arabic version, how can I fix this problem?

**Comment by 10110111** [comment]
You need to be more specific what your problem is.

**Reply by Mohammed1948** [reply]
I downloaded the program, and I want to run it in Arabic, but the Arabic language option is missing or has a problem!

**Reply by gzotti** [reply]
Is the Arab language missing (is there no entry in the language selection dropdown), or does it have a problem (you set Arab but it displays in another language, or a font is missing, or only a few characters are missing, or right-to-left is broken, ...) ? On what platform? The issue report template includes hints on what we need as minimum kind of input from your side to be able to start identifying Issues. This forum is for usage questions.

**Reply by Mohammed1948** [reply]
there is no entry in the language selection dropdown.
on Windows

**Comment by gzotti** [comment]
If it does not look like this:
![stellarium-082](https://github.com/Stellarium/stellarium/assets/18099873/6f91a67f-9e5c-4e0a-9cc9-430bd551255f)

I hope some Arab-speaking user knows what to do.

**Reply by Mohammed1948** [reply]
No, the problem appears when downloading, as the "Arabic language" option is not in the list!

**Reply by gzotti** [reply]
You download any of the available Windows versions and set whichever supported language you need inside the program. Also the website is translated to Arab, so I don't understand in which list an entry for Arab is missing. Can you illustrate what you are doing?

**Reply by Mohammed1948** [reply]
![Capture](https://github.com/Stellarium/stellarium/assets/134861392/34e80507-94ad-42ae-9318-c7c9bf463385)

**Reply by alex-w** [reply]
InnoSetup installer hasn't Arabic translation

**Reply by Mohammed1948** [reply]
I Know....but why ?!

**Reply by gzotti** [reply]
Probably just nobody has translated it. After installation, the program is available in a large number of languages.

**Reply by alex-w** [reply]
> I Know....but why ?!

Please ask it in InnoSetup project resources, because this is separate project and Stellarium is not related to it.

As far as I know InnoSetup has[ unofficial Arabic translation](https://raw.github.com/jrsoftware/issrc/main/Files/Languages/Unofficial/Arabic.isl) and we may add it into Stellarium installer, but unofficial translations are of unknown translation quality - I don't know Arabic language and I cannot say how this translation is good and acceptable.
